aptos-move/framework/src/module_metadata.rs

```diff
@@ -148,31 +149,64 @@ impl KnownAttribute {
     }
 }
 
+const METADATA_CACHE_SIZE: usize = 1024;
```

Comment:
How did you come up with this magic number?

Reply:
the epilogue pulls in a pretty large chunk of the framework, and it's 70-ish modules. 1024 would be plenty of modules for a good hit rate.